syserol/figures/common.py

```diff
@@ -269,26 +269,82 @@ def buildFigure3(legends=True, heatmap=False):
         for aa in ax:
             aa.axis('equal')
     else:
-        ax, f = getSetup((8, 8), (1, 4))
-        
-        cbar_kws = {"orientation":"horizontal"}
+        f = plt.figure(figsize=(10, 15))
+        gs = gridspec.GridSpec(2, 4,
+                            width_ratios=[2, 25, 12, 25],
+                            height_ratios=[1, 1],
+                            wspace=0, hspace=.3
+                            )
```

Comment:
`common.py` should only have code used for multiple figures. Move this to the Fig. 2 file.

Reply:
I believe this is for figure 3, and since the function that builds figure 3 is in common, I put it there. Don't remember why it was there in the first place, but should I move the whole function over?

Reply:
Yeah, it's not you who put it here, and it makes sense that it was here because it used to be split across figures. Now, it should go into Fig. 3. Please move it.